code/modules/admin/admin_verbs.dm

```diff
@@ -714,3 +717,18 @@ GLOBAL_LIST_INIT(admin_verbs_hideable, list(
 
 	log_admin("[key_name(usr)] has [AI_Interact ? "activated" : "deactivated"] Admin AI Interact")
 	message_admins("[key_name_admin(usr)] has [AI_Interact ? "activated" : "deactivated"] their AI interaction")
+
+
+/client/proc/give_faction_objective()//ST13 debugging, gives selected faction an objective
+	set name = "Give Faction Objective"
+	set category = "Debug"
+
+	var/datum/faction/I = input("Select a faction.", "Add Objective") as null|anything in SSfaction.factions
+	if(!I)
+		return
+	var/datum/factionobjective/A = input("Select an objective to assign to [I.name].", "Add Objective") in list(/datum/factionobjective/destroy1, "cancel")
```

Comment:
Just do the null| thing you did above, that adda the cancel and means you only need to do if(!A) to check for it

Reply:
I tried to do it as null|"/datum/ect", but It gave an error so I settled with this. I might've done it wrong, though.